css/index.css

```diff
@@ -1 +1,379 @@
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+* {
+	box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%;
+}
+
+@media (max-width: 800px) {
+	.slogan-section img {
+		display: none;
+	}
+	
+	header {
+		display: flex;
+		flex-direction: column-reverse;
+		width: 100%;
+	}
+	header .logo {
+		align-self: center;
+	}
+	header nav  {
+		display: flex;
+		justify-content: space-around;
+	
+	}
+}
+
+@media (max-width: 500px) {
+	header {
+		display: flex;
+		flex-direction: column-reverse;
+		width: 100%;
+		border-top: 1px gray solid;
+		
+
+	}
+	header nav {
+		display: flex; 
+		flex-direction: column;
+		align-items: center;
+		/*margin: 10px 0; */
+		border-top: 1px gray solid;
+		padding: 15px 0;
+		padding-top: 0;
+		margin-top: 0;
+		
+	}
+	header nav a {
+		width: 100%;
+		border-bottom: 1px gray solid;
+		align-self: center;
+		justify-items: center;
+		text-align: center;
+		padding: 1.5rem 0px;
+		
+		
+	}
+	header .logo {
+		padding: 5px 0;
+		
+	}
+	body {
+		display: flex;
+		flex-direction: column;
+	}
+	
+	body .inner-body {
+		display: flex;
+		flex-direction: column;
+	}
+}
+
+body {
+	background: rgba(5, 159, 255, .6);
+	margin: auto;
+	overflow: hidden;
+
+
+}
+.body-container {
+	max-width: 880px;
+	margin: 0 auto;
+	background: white;
+}
+.services .body-container {
+	width: 60%;
+	margin: 0 auto;
+	background: white;
+}
+header {
+	width: 100%;
+	display: flex;
+	height: auto;
+	position: relative;
+	margin-bottom: 15px;
+	justify-content: space-between;
+}
+
+header hr {
+	display: none;
+}
+nav { 	
+	background: white;
+	display: flex;
+	position: relative;
+	justify-content: space-around; 
+	margin: 0 auto;
+	width: 100%;
+	padding: 15px 1.5rem;
+	
+
+}
+
+nav a {
+	text-decoration: none;
+	color: gray;
+	font-size: 1.6rem;
+	width: 16%;
+}
+
+header nav .logo {
+	height: auto;
+	display: inline-block;	
+}
+
+header nav .logo img {
+	height: auto;
+	width: 100%;
+	align-items: baseline;
+}
+
+
+
+.services-header-img {
+	width: 100%;
+	margin-bottom: 10px;
+}
+
+.services-header {
+	margin: 20px auto;
+}
+
+.contact-header {
+	background-image: url('img/services-header.jpg');
+	width: 100%;
+	height: 200px;
+	display: flex;
+
+}
+
+.contact-header h1 {
+	justify-content: center;
+	font-size: 10rem;
+	text-align: center;
+	margin: 5px auto;
+}
+.slogan-section {
+	
+	width: 100%;
+	display: flex;
+	justify-content: space-around;
+	height: 20%;	
+	align-items: center;
+	padding-bottom: 20px;
+	margin-bottom: 20px;
+}
+
+.slogan-section .box {
+	height: 100%; 
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	width: 48%;
+	justify-items: center;
+
+}
+
+.slogan-section img {
+	width: 40%;
+	height: auto;
+	justify-items: center;
+}
+
+.box .home-button {
+	border: 2px solid black;
+	background: white;
+	font-size: 1.6rem;
+	padding: 10px 1rem;
+	width: 75%;
+}
+
+
+.inner-body {
+display: flex;
+flex-direction: row;
+margin: 10px 1rem;
```

Comment:
remember, when using rem on box model, it will change when the user changes their default font-size. make sure you want that behavior. often times you won't